File: test-smtp.js
const nodemailer = require("nodemailer");

async function main() {
  let transporter = nodemailer.createTransport({
    host: "smtp-mail.outlook.com",
    port: 587,
    secure: false, // true for 465, false for other ports
    auth: {
      user: "MBAplanner@outlook.com",
      pass: "byfxigwgfnwvppyf",
    },
  });

  try {
    let info = await transporter.sendMail({
      from: '"MBA Planner Test" <MBAplanner@outlook.com>',
      to: "tarun.shekhawat2027@bitsom.edu.in",
      subject: "Test Email from SMTP Setup",
      text: "If you are reading this, your Outlook SMTP is working!",
      html: "<b>If you are reading this, your Outlook SMTP is working!</b>",
    });
    console.log("Message sent successfully: %s", info.messageId);
  } catch (error) {
    console.error("Error occurred while sending email:");
    console.error(error);
  }
}

main();

File: test-smtp.py
import smtplib
from email.mime.text import MIMEText

msg = MIMEText('If you are reading this, your Outlook SMTP is working!')
msg['Subject'] = 'Test Email from SMTP Setup'
msg['From'] = 'MBAplanner@outlook.com'
msg['To'] = 'tarun.shekhawat2027@bitsom.edu.in'

try:
    server = smtplib.SMTP('smtp-mail.outlook.com', 587)
    server.set_debuglevel(1)
    server.starttls()
    server.login('MBAplanner@outlook.com', 'byfxigwgfnwvppyf')
    server.send_message(msg)
    server.quit()
    print("Email sent successfully!")
except Exception as e:
    print(f"Failed to send email: {e}")
